src/internal.rs

```diff
@@ -14,10 +14,14 @@ use std::time;
 
 use exec::CommandTemplate;
 use lscolors::LsColors;
-use regex::RegexSet;
+use regex::{Regex, RegexSet};
 use regex_syntax::hir::Hir;
 use regex_syntax::Parser;
 
+lazy_static! {
+    static ref SIZE_CAPTURES: Regex = { Regex::new(r"^(\+|-)(\d+)([a-zA-Z]{1,2})$").unwrap() };
```

Comment:
Why do we need two regex patterns for capturing and validation? Wouldn't one be enough?